velox/expression/VectorFunction.h

```diff
@@ -143,7 +143,8 @@ std::optional<std::vector<FunctionSignaturePtr>> getVectorFunctionSignatures(
 std::shared_ptr<VectorFunction> getVectorFunction(
     const std::string& name,
     const std::vector<TypePtr>& inputTypes,
-    const std::vector<VectorPtr>& constantInputs);
+    const std::vector<VectorPtr>& constantInputs,
+    const TypePtr resultType = nullptr);
```

Comment:
The `resultType` should ideally be required here for the factory input argument to be unambiguous.
Let's remove the default.

Reply:
Or default to an `Unknown` Type if needed.